src/app/OperationalDeviceProxy.cpp

```diff
@@ -42,6 +42,18 @@ using namespace chip::Callback;
 
 namespace chip {
 
+void OperationalDeviceProxy::MoveToState(State aTargetState)
+{
+    if (mState != aTargetState)
+    {
+        ChipLogDetail(Controller,
+                      "OperationalDeviceProxy[CF" ChipLogFormatX64 ":" ChipLogFormatX64 "]: Moving from state %d --> %d",
```

Comment:
Why not a dash instead of colon between compressed fabric ID and node ID, since that is the operatinoal instance name?  Also, what is CF?

```suggestion
                      "OperationalDeviceProxy[CF" ChipLogFormatX64 "-" ChipLogFormatX64 "]: Moving from state %d --> %d",
```

I'm also wondering if we should log state transitions, even if there is not a new state.  Would it be intentional that we should be calling MoveToState often to the state we're already in?  If we see that type of logging, is it possible this is indicative of a bug, and not just noise?